### ISSUE TITLE
Compatibility to ARM Cortex M0, M3, esp8266?

### ISSUE THREAD

**Issue by dsyleixa** [avandalen/avdweb_FreqPeriodCounter#2]
Is your lib also compatible for Arduino Due (M3), Teensy, M0 / Zero, esp8266? Because the AVRs are actually completely obsolete nowadays, because too weak both of cpu clock and RAM. e.g., the Due calculates  Float already 3x as fast as an AVR (Uno, Mega), and even up to 150x as fast with Integer.
So frequency polling could be achieved much faster and much more accurately by ARM cpus.

**Comment by avandalen** [comment]
Yes it works for the Arduino Zero (SAMD21), the rest must be tested

From: dsyleixa 
Sent: Monday, January 22, 2018 1:38 PM
To: avandalen/avdweb_FreqPeriodCounter 
Cc: Subscribed 
Subject: [avandalen/avdweb_FreqPeriodCounter] Compatibility to ARM Cortex M0, M3, esp8266? (#2)

Is your lib also compatible for Arduino Due (M3), Teensy, M0 / Zero, esp8266? Because the AVRs are actually completely obsolete nowadays, because too weak both of cpu clock and RAM. e.g., the Due calculates Float already 3x as fast as an AVR (Uno, Mega), and even up to 150x as fast with Integer.
So frequency polling could be achieved much faster and much more accurately by ARM cpus.
